Source/WebKit/Configurations/GPUService.xcconfig

```diff
@@ -25,6 +25,7 @@
 
 PRODUCT_NAME = com.apple.WebKit.GPU;
 PRODUCT_BUNDLE_IDENTIFIER = $(PRODUCT_NAME);
+PRODUCT_BUNDLE_IDENTIFIER[sdk=iphoneos17.4*] = $(PRODUCT_NAME)Service;
```

Comment:
Is this needed? If setting EXCLUDED_SOURCE_FILE_NAMES does successfully prevent us from generating the product, I don't see why we would need to customize the bundle ID, because there will be no Info.plist to write it to.

Reply:
Yes, this appears to be needed, since the Info.plist is still being created.